### ISSUE TITLE
Jobs output not loading

### ISSUE THREAD

**Issue by lilapetryshyn** [ansible/tower-operator#934]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
After the AWX job completed, job output is not available and looks like that:
<img width="1093" alt="Screenshot 2022-06-01 at 16 42 11" src="https://user-images.githubusercontent.com/32909547/171419376-74afe14f-0b44-40c6-b588-a8f23d5c4f68.png">

During job running, we can see and follow all the logs, but just right after job completed it becomes like this.
The only way to check log output is to download it.

Did anybody face with the same problem maybe?

##### ENVIRONMENT
* AWX version: 20.0.1
* Operator version: 0.18.0
* Kubernetes version: 1.21
* AWX install method: AWS EKS cluster with Fargate managed nodes

**Comment by rakesh561** [comment]
we did see this this type of blank output for few jobs that succeeded and also for the one that failed. It was because of K8s trying to archive the logs once that is done next jobs were showing output.

**Comment by lilapetryshyn** [comment]
I also noticed error in awx-task container, that might be related, in `/var/log/tower/dispatcher.log`:
```
 ERROR    [0fe15cbc5bdd4c62a78da0885a226713] awx.main.dispatch PID:465 Worker failed to run task awx.main.tasks.system.purge_old_stdout_files(*[], **{}
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/dispatch/worker/task.py", line 90, in perform_work
    result = self.run_callable(body)
  File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/dispatch/worker/task.py", line 65, in run_callable
    return _call(*args, **kwargs)
  File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/tasks/system.py", line 325, in purge_old_stdout_files
    for f in os.listdir(settings.JOBOUTPUT_ROOT):
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/awx/job_status'
```
i've seen issue already created for this one https://github.com/ansible/awx/issues/11903 , but no updates lately

**Comment by fosterseth** [comment]
duplicate of https://github.com/ansible/awx/issues/12194
